src/js/components/RadioButtonGroup/index.d.ts

```diff
@@ -5,8 +5,8 @@ export interface RadioButtonGroupProps {
   disabled?: boolean;
   name: string;
   onChange?: ((event: React.ChangeEvent<HTMLInputElement>) => void);
-  options: (string | { disabled?: boolean, id?: string, label?: (string | React.ReactNode), value: string})[];
-  value?: string | object;
+  options: (string | { disabled?: boolean, id?: string, label?: (string | React.ReactNode), value: string | number })[];
+  value?: string | number | object;
```

Comment:
Should we remove the `object` option?

Reply:
nope, we still allow options to contain objects. But, I reviewed the typescript types again and found another issue, fixed.

Reply:
In this case, should it be added to PropTypes in doc.js?

Reply:
update to add value as number to prop types